solidity/contracts/system/KeepFactorySelection.sol

```diff
@@ -49,7 +49,7 @@ library KeepFactorySelection {
     function initialize(
         Storage storage _self,
         IBondedECDSAKeepFactory _defaultFactory
-    ) internal {
+    ) public {
```

Comment:
How come?

Reply:
Oh, I forgot to flag this change. Does it make sense to keep it `internal`? After this change, we have all library functions called from the contract `public` so they are not copied anywhere and stay in the library. All functions used only from the library are `internal`.

Reply:
Oh very good, sorry, I forgot this is a library. That's fine.